[docs] Edit info about Trivy NS scanning (#15565)
deckhouse/deckhouse: docs/documentation/pages/admin/configuration/security/SCANNING.md

## Vulnerability scanning

> DKP scans of all container images used in cluster Pods.
> The checks for:

Reviewer: ```suggestion
DKP scans all container images used by cluster pods.
The checks include:
```
